### PR TITLE
#409 Allow project leads to check description bullets

### LINKED ISSUE #409: Work Package Progress - Update Desc Bullet Checking Permissions
### Description

From James:

> permissions issue with checking off bullets: Codebase says bullets can only be checked off if someone is a LEADERSHIP (heads) or above and this means leads can’t check bullets off. I think leads (MEMBER role) who are assigned as the project lead for the project should be allowed to check bullets.

### Acceptance Criteria

change it to match the criteria set by james. there should be a permissions functions in the desc bullets utils files in the backend or something

### Proposed Solution

.

### Mocks

.

### PR DESCRIPTION
**Pull request by Peyton-McKee**
## Changes

Added another check to see if the project lead is editing the description bullets even if they are a member 

## Test Cases

Case A: Member who is project lead checking description bullet 

Screen Shot 2022-11-07 at 4 59 30 PM

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please review the [contributor guide](https://nerdocs.atlassian.net/wiki/spaces/NER/pages/8060929/Software+Contributor+Guide) and reach out to your Tech Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [x] All commits are tagged with the ticket number
- [x] No linting errors / newline at end of file warnings
- [x] All code follows repository-configured prettier formatting
- [x] No merge conflicts
- [x] All checks passing
- [x] Screenshots of UI changes (see Screenshots section)
- [x] Remove any non-applicable sections of this template
- [x] Assign the PR to yourself
- [x] No `yarn.lock` changes (unless dependencies have changed)
- [x] Request reviewers & ping on Slack
- [x] PR is linked to the ticket (fill in the closes line below)

Closes #409  (issue #)

### CODE REVIEW

File: src/backend/src/utils/description-bullets.utils.ts [hunk 1]
```diff
@@ -19,14 +19,28 @@ export const hasBulletCheckingPermissions = async (userId: number, descriptionId
...
   if (!descriptionBullet) return false;
 
   if (!user) return false;
 
-  if (user.role === Role.APP_ADMIN || user.role === Role.ADMIN || user.role === Role.LEADERSHIP) {
+  const leader =
+    descriptionBullet.workPackageDeliverables?.wbsElement.projectLead ||
+    descriptionBullet.workPackageExpectedActivities?.wbsElement.projectLead;
+
+  console.log(leader);
```
Comment (Ari-S-123) [inline src/backend/src/utils/description-bullets.utils.ts:37]: Should this console.log still be here?

**Review by Ari-S-123 — CHANGES_REQUESTED** [review]

File: src/backend/src/utils/description-bullets.utils.ts (hunk 1 above)
Comment (Peyton-McKee) [reply src/backend/src/utils/description-bullets.utils.ts]: u didnt see anything

**Review by Ari-S-123 — DISMISSED** [review]

**Review by anthonybernardi — CHANGES_REQUESTED** [review]
can u also add the project manager pls

**Review by anthonybernardi — CHANGES_REQUESTED** [review]
definitely not prettier formatted

**Review by anthonybernardi — APPROVED** [review]

**Pull request merged**